source/core/src/core/bronze/infrastructure/config/__init__.py

```diff
@@ -0,0 +1,3 @@
+from core.bronze.infrastructure.config.table_names import TableNames as BronzeTableNames
+
+__all__ = ["BronzeTableNames"]
```

Comment:
```suggestion
from core.bronze.infrastructure.config.table_names import TableNames

__all__ = ["TableNames"]
```

no BronzeTableNames :D